contribs/gmf/src/controllers/AbstractAppController.js

```diff
@@ -591,6 +561,7 @@ export function AbstractAppController(config, map, $scope, $injector) {
   /**
    * @export
    */
+  // @ts-ignore: todo
```

Comment:
todo?

Reply:
Yes, todo.  I suspect we don't want to leave this like this forever.  If you have a better idea for a comment, please let met know what to write.

Reply:
```suggestion
  // @ts-ignore: wanted to define new property on window
```
:-)

Reply:
Thanks for the suggestion.  I'll apply them, hold on...